test/unit/lib/addMessages.js

```diff
@@ -29,6 +31,31 @@ describe('addMessages', () => {
     expect(openServices[0].openingTimesMessage).to.be.equal('Open 24 hours');
   });
 
+  it('should return the opening times message and open state when between 12:00am and 01:00am Bristish Summer Time', () => {
+    const spanningSundayMidnightOrg = {
+      openingTimes: {
+        general: {
+          monday: [{ opens: '0:00', closes: '20:00' }, { opens: '23:00', closes: '23:59' }],
```

Comment:
I'm pretty sure it doesn't matter but I think the first opens should be `00:00` for consistency.